<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-CLFL9AL" failures="1" tests="8" name="SPPersonnelandARMeghalaya.Duplicatetestcase" time="16.071" errors="0" timestamp="2024-04-12T00:35:51 IST" skipped="0">
  <testcase classname="SPPersonnelandARMeghalaya.Duplicatetestcase" name="allservicesnavigation" time="6.701"/>
  <system-out/>
  <testcase classname="SPPersonnelandARMeghalaya.Duplicatetestcase" name="Deptnavigation" time="5.315"/>
  <system-out/>
  <testcase classname="SPPersonnelandARMeghalaya.Duplicatetestcase" name="LogocheckTestCases" time="0.158"/>
  <system-out/>
  <testcase classname="SPPersonnelandARMeghalaya.Duplicatetestcase" name="ServicecountTestCases" time="0.120"/>
  <system-out/>
  <testcase classname="SPPersonnelandARMeghalaya.Duplicatetestcase" name="duplicateTestCases" time="0.159"/>
  <system-out/>
  <testcase classname="SPPersonnelandARMeghalaya.Duplicatetestcase" name="servicecountfromexcel" time="0.306">
    <failure message="The following asserts failed:
	&lt;table border=&#039;1&#039; style=&#039;width: 100%;&#039;&gt;&lt;tr&gt;&lt;th style=&#039;padding: 0; background-color: #48C9B0; border: 1px solid black;&#039;&gt;Found extra service name in UI but absent in excel:&lt;/th&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td style=&#039;padding: 0; background-color: #E8F8F5 ;text-align: left; border: 1px solid black;&#039;&gt;&lt;ul style=&#039;list-style-type: none; padding: 0; margin: 0;&#039;&gt;&lt;li style=&#039;padding: 5px; margin: 0; border-bottom: 1px solid black;&#039;&gt;Return of Assests and Liabilities under Meghalaya Public Service Rules 2016&lt;/li&gt;&lt;/ul&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/table&gt; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
<table border='1' style='width: 100%;'><tr><th style='padding: 0; background-color: #48C9B0; border: 1px solid black;'>Found extra service name in UI but absent in excel:</th></tr><tr><td style='padding: 0; background-color: #E8F8F5 ;text-align: left; border: 1px solid black;'><ul style='list-style-type: none; padding: 0; margin: 0;'><li style='padding: 5px; margin: 0; border-bottom: 1px solid black;'>Return of Assests and Liabilities under Meghalaya Public Service Rules 2016</li></ul></td></tr></table></table> expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at PageObjects.commonlocators.servicecountfunction(commonlocators.java:194)
at SPPersonnelandARMeghalaya.Duplicatetestcase.servicecountfromexcel(Duplicatetestcase.java:117)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- servicecountfromexcel -->
  <system-out/>
  <testcase classname="SPPersonnelandARMeghalaya.Duplicatetestcase" name="InformationTestCases" time="3.084"/>
  <system-out/>
  <testcase classname="SPPersonnelandARMeghalaya.Duplicatetestcase" name="IMbacktodirectory" time="0.228"/>
  <system-out/>
</testsuite> <!-- SPPersonnelandARMeghalaya.Duplicatetestcase -->
